epub32/spec/epub-contentdocs.html

```diff
@@ -681,8 +681,8 @@ <h4>Content Conformance</h4>
 						<dl class="conformance-list">
 							<dt id="math-pres">Presentation MathML</dt>
 							<dd><p id="confreq-mathml-pres">The <code>math</code> element MUST contain only <a
-										href="https://www.w3.org/TR/MathML3/chapter3.html">Presentation MathML</a>, with
-									the exception of the <code>annotation-xml</code> element as defined below.</p></dd>
```

Comment:
shouldn't we keep something along the lines of 

> with the exception of annotation content within the `annotation-xml` element

?

Reply:
I don't think it's necessary to say, is it? annotation-xml is a presentation mathml element. Now that there are no restrictions on it, there's nothing more to say. The restriction is still required to deal with content mathml, as described in the next section, plus you can't just embed arbitrary xml inside the math element.

Reply:
yeah it was just to avoid any confusion with:

> MUST contain only presentation MathML

If `annotation-xml` is used there can be non-Presentation content in the descendants. But maybe it's just clear enough as is…

Reply:
I honestly can't get worked up about this one. If you think it's clearer, let's just add it back.